documentation/book/con-topic-operator-cluster-label.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// assembly-deploying-the-topic-operator.adoc
+
+[id='con-topic-operator-cluster-label-{context}']
+
+= Identifying a Kafka cluster for topic handling
+
+A `KafkaTopic` resource includes a label that defines the appropriate name of the Kafka cluster (derived from the name of the `Kafka` resource) to which it belongs.
+
+[source,yaml,subs="attributes+"]
+----
+  labels:
+    strimzi.io/cluster: my-cluster
+----
+
+The label is used by the Topic Operator to identify the `KafkaTopic` resource and create a new topic.
```

Comment:
It's not just about creation but handling in general.

Reply:
The label is used by the Topic Operator to identify the `KafkaTopic` resource and create a new topic, and also in subsequent handling of the topic.